packages/@uppy/dashboard/src/components/PanelTopBar.js

```diff
@@ -7,6 +7,8 @@ function DashboardContentTitle (props) {
 }
 
 function PanelTopBar (props) {
+  const notOverFileLimit = props.maxNumberOfFiles !== props.totalFileCount
```

Comment:
`props.totalFileCount < props.maxNumberOfFiles` makes the intent a bit more clear I think. And a check to see if maxNumberOfFiles even exists?

Reply:
Updated, like this?

Reply:
yea that makes more sense to me :+1: